docs/content/how-to/mce/Dual/hostedcluster/hostedcluster.md

```diff
@@ -0,0 +1,248 @@
+In this section we will focus in all the related objects which they will be necessary to achieve a Disconnected Hosted Cluster deployment.
+
+**Premises**:
+
+- HostedCluster Name: `hosted-dual`
+- HostedCluster Namespace: `clusters`
+- Disconnected: `true`
+- Network Stack: `Dual`
+
+## Openshift Objects
+
+### Namespaces
+
+The operator in a usual situation will be in charge of create the HCP (HostedControlPlane) namespace, but in this case we want to include all the objects prior the operator to reconcicle over the HostedCluster object. This way when the operator begins the reconcilliation, it will find all the objects in place.
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  creationTimestamp: null
+  name: clusters-hosted-dual
+spec: {}
+status: {}
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  creationTimestamp: null
+  name: clusters
+spec: {}
+status: {}
+```
+
+We will not create object by object but, concatenante all of them in the same file and apply them with just 1 command.
+
+### ConfigMap and Secrets
+
+These are the ConfigMaps and Secrets which we will include in the HostedCluster deployment.
+
+```yaml
+---
+apiVersion: v1
+data:
+  ca-bundle.crt: |
+    -----BEGIN CERTIFICATE-----
+    -----END CERTIFICATE-----
+kind: ConfigMap
+metadata:
+  name: clusters-hosted-dual-ca
+  namespace: clusters
+---
+apiVersion: v1
+data:
+  .dockerconfigjson: xxxxxxxxx
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: hosted-dual-pull-secret
+  namespace: clusters
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sshkey-cluster-hosted-dual
+  namespace: clusters
+stringData:
+  id_rsa.pub: ssh-rsa xxxxxxxxx
+---
+apiVersion: v1
+data:
+  key: nTPtVBEt03owkrKhIdmSW8jrWRxU57KO/fnZa8oaG0Y=
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: hosted-dual-etcd-encryption-key
+  namespace: clusters
+type: Opaque
+```
+
+### RBAC Roles
+
+This is not mandatory, but it allow us to have the Assisted Service Agents located in the same HostedControlPlane namespace than the HostedControlPlane and still be managed by CAPI.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: capi-provider-role
+  namespace: clusters-hosted-dual
+rules:
+- apiGroups:
+  - agent-install.openshift.io
+  resources:
+  - agents
+  verbs:
+  - '*'
+```
+
+### Hosted Cluster
+
+This is the HostedCluster Object
+
+```yaml
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  name: hosted-dual
+  namespace: clusters
+spec:
+  additionalTrustBundle:
+    name: "clusters-hosted-dual-ca"
+  olmCatalogPlacement: guest
+  imageContentSources:
```

Comment:
Do we need this here? This is for mirror registry sources for workloads on the guest cluster rather than part of the setup of the hosted cluster.

Reply:
I think that it's worth to mention it. I can specify it in the documentation itself.